mod/posts/set/all/post.rb

```diff
@@ -0,0 +1,9 @@
+format :html do
+  view :posts_tab do
+    output [field_nest(:post, items: { view: :thin_listing }),
+            link_to("Add Post", path: { mark: :post,
+                                        action: :new,
+                                        "_#{card.type_name}": card.name },
+                                class: "btn btn-primary")]
+  end
```

Comment:
this post tab is here because it's the same for all types.  We might be able to repeat something like this pattern for other types.